src/runtime/runtime_unix.go

```diff
@@ -256,3 +256,41 @@ func growHeap() bool {
 	setHeapEnd(heapStart + heapSize)
 	return true
 }
+
+func init() {
+	// Set up a channel to receive signals into.
+	// A channel size of 1 should be sufficient in most cases, but using 4 just
+	// to be sure.
+	signalChan = make(chan uint32, 4)
```

Comment:
Could you elaborate on that? Could you tell me what the requirements/context are for this?